cmd/ronin/dbcmd.go

```diff
@@ -243,8 +250,86 @@ WARNING: This is a low-level operation which may cause database corruption!`,
 		},
 		Description: "Exports the specified chain data to an RLP encoded stream, optionally gzip-compressed.",
 	}
+	dbInspectEnodeDBCmd = &cli.Command{
+		Action: dbInspectEnodeDB,
+		Name:   "inspect-enodedb",
+		Usage:  "Inspect nodes in enode db",
+		Flags: []cli.Flag{
+			utils.DataDirFlag,
+			&cli.StringFlag{
+				Name:  "peersfile",
+				Usage: "File containing a list of peers from admin.peers",
+			},
+		},
+		Category: "DATABASE COMMANDS",
+	}
 )
 
+func dbInspectEnodeDB(ctx *cli.Context) error {
+	path := ctx.String(utils.DataDirFlag.Name)
+	db, err := enode.OpenDB(path)
```

Comment:
Seems like this is path to nodes directory inside the datadir not the datadir, right? So the datadir flag in this command will have different meaning from other commands, this might be confusing.

Reply:
It's path for nodes folder not chain datadir, yes maybe we can create different flag for it.